herd/tests/instructions/AArch64/A26.litmus

```diff
@@ -0,0 +1,9 @@
+AArch64 A26
+(* Tests load exclusive register, symbolic location, no offset*)
+{ 0:X1=x}
```

Comment:
Location `x` _must_ hold a 64 bits wide integer (and `0:X0` is better printed as such):
```suggestion
{ 0:X1=x; int64_t x; int64_t 0:X0; }
```